static/js/components/UserChip.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React from 'react';
+import { Card } from 'react-mdl/lib/Card';
+import Icon from 'react-mdl/lib/Icon';
+
+import { getPreferredName, getEmployer } from '../util/util';
+import ProfileImage from './ProfileImage';
+import { mstr } from '../util/sanctuary';
+import type { Profile } from '../flow/profileTypes';
+
+const UserChip = ({ profile }: {profile: Profile}): React$Element<*> => (
+  <Card className="user-chip" shadow={2}>
+    <div className="profile-info">
+      <span className="name">
+        { getPreferredName(profile) }
+      </span>
+      <span className="employer">
+        { mstr(getEmployer(profile)) }
+      </span>
+      <a href={`/users/${profile.username}`} className="mm-minor-action">
+        <Icon name="person" />
```

Comment:
this works:

```
// in UserChip.js
      <a href={`/users/${profile.username}`} className="mm-minor-action">
        <Icon name="person" />
        <span>View profile</span>
      </a>

// in user_chip.scss
    a:hover {
      text-decoration: none;
    }

    a:hover > span {
      text-decoration: underline;
    }
```

just a bit hacky as it wraps the text in an otherwise-unnecessary <span>